### PR TITLE
[MRG] Add EMG channel and fix droppped non-EEG channels

### LINKED ISSUE #107: non-eeg channels are dropped from eeg files
I noticed that the EMG channel type wasn't included in the list of types, so I was going to make a nice easy PR to fix it...
**But** then, as I was writing a test I discovered that if I set the type of channel to EMG in an eeg file (was testing edf data since part of the test already modifies a channel so I figured I'd piggyback off that), this channel is dropped.
I narrowed the reason down to the `infer_eeg_placement_scheme` function, which will drop any non-eeg channels because of `raw.pick`.
I can fix this, but I was wondering whether this was in fact the intended behaviour or not?

Now that I think about it, I am pretty sure if there is MEG data which contains EEG data, this function will actually remove all MEG channel info, rendering the data completely wrong, so I guess that this behaviour is indeed unintended.

### PR DESCRIPTION
**Pull request by monkeyman192**
closes #107

### CODE REVIEW

File: mne_bids/tests/test_mne_bids.py [hunk 1]
```diff
@@ -176,6 +178,15 @@ def test_edf():
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
 
+    # ensure there is an EMG channel in the channels.tsv:
+    channels_tsv = make_bids_filename(
+        subject=subject_id, session=session_id, task=task, run=run,
+        suffix='channels.tsv', acquisition=acq,
+        prefix=op.join(output_path, 'sub-01/ses-01/eeg'))
+    if op.exists(channels_tsv):
+        df = pd.read_csv(channels_tsv, sep='\t')
```
Comment (sappelhoff) [inline mne_bids/tests/test_mne_bids.py]: why do you have the `op.exists` if clause? Wouldn't the channels.tsv always exist in this test case?

File: mne_bids/tests/test_mne_bids.py [hunk 2]
```diff
@@ -135,12 +132,8 @@ def test_bti():
 
     assert op.exists(op.join(output_path, 'participants.tsv'))
 
-    # FIXME: see these issues for reference:
```
[the remark this answers is not in the document]
Comment (sappelhoff) [reply mne_bids/tests/test_mne_bids.py]: mmmh curious ... the original issue is still open, but there was a related fix here: https://github.com/bids-standard/bids-validator/pull/607/files

... perhaps this one fixed the other one by accident :-) ... although BTi data still is not fixed.

**Comment by sappelhoff** [comment]
> as far as I can tell neither of these test failures are my fault?

your changes look good to go from my side. The AppVeyor and CircleCI issues seem related ... but I don't know what bothers them. Let's wait a day and then rebuild, perhaps it's some issue with pip or conda or the services or ???

**Comment by jasmainak** [comment]
Merged. Thanks @monkeyman192 !

it's probably one of our dependencies in `environment.yml`. @sappelhoff do you have time to look into it?

**Comment by sappelhoff** [comment]
yes, fixed in #104

**Pull request merged**